### PR TITLE
Added svgo, to use in building asc-assets

### PR DESCRIPTION
**Pull request by chrisvanmook**

### CODE REVIEW

File: packages/asc-assets/.svgo.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+plugins:
+  - removeViewBox: false
+  - removeDimensions: true
```
Comment (janjaap) [inline packages/asc-assets/.svgo.yml]: There is a lot more optimisations we could potentially benefit from, like:
```yaml
- removeDoctype
- removeComments
- removeMetadata
- removeXMLNS
- removeEditorsNSData
- cleanupIDs
- removeRasterImages
- removeUselessDefs
- removeUnknownsAndDefaults
- removeUselessStrokeAndFill
- removeHiddenElems
- removeEmptyText
- removeEmptyAttrs
- removeEmptyContainers
- removeUnusedNS
- removeDesc
- prefixIds
- ...
```

File: packages/asc-assets/package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "assets": "assets"
   },
   "scripts": {
-    "build": "mkdir -p lib && cp -R src/* lib/"
+    "build": "mkdir -p lib && cp -R src/* lib/ && svgo --config=.svgo.yml -f ./src/Icons/ -o ./lib/Icons"
```
Comment (janjaap) [inline packages/asc-assets/package.json]: Why not use SVGO in a Webpack loader?

File: packages/asc-core/src/components/Icon/Icon.ts [hunk 3]
```diff
@@ -15,8 +15,6 @@ const Icon = styled.span<Props>`
   height: 20px;
 
   & > svg {
-    width: 20px;
```
Comment (janjaap) [inline packages/asc-core/src/components/Icon/Icon.ts]: Is this change related to adding SVGO?

File: packages/asc-assets/package.json (hunk 2 above)
Comment (chrisvanmook) [reply packages/asc-assets/package.json]: Because we don't create a bundle, we only provide optimized svg's in the package. It's up to the user of asc-assets (developer) how to bundle the code with the svg's.

File: packages/asc-core/src/components/Icon/Icon.ts (hunk 3 above)
Comment (chrisvanmook) [reply packages/asc-core/src/components/Icon/Icon.ts]: Yes and no. It's not directly linked to the svgo, however, I noticed while playing with svgo config, these lines could be removed as they don't do anything (because now we get rid of the width & height attributes)

Comment (janjaap) [reply packages/asc-core/src/components/Icon/Icon.ts]: Nice

File: packages/asc-assets/.svgo.yml (hunk 1 above)
Comment (chrisvanmook) [reply packages/asc-assets/.svgo.yml]: I didn't really dove into this in detail yet. I only added the optimisations I knew we would need for now. But would be great if we could use these! Do you have any experience with the optimisations you listed above?

Comment (janjaap) [reply packages/asc-assets/.svgo.yml]: Most of them, yes.
